entities/entities/darkrp_laws/cl_init.lua

```diff
@@ -2,10 +2,16 @@ include("shared.lua")
 
 local Laws = {}
 
+ENT.DrawPos = Vector(1, -111, 58)
+
+local color_navy_200 = Color(0, 0, 70, 200)
+local color_red = Color(255, 0, 0, 255)
+local color_white = color_white
```

Comment:
This works for me, but where is it defined? I'm not sure we can depend on it.

Reply:
This is defined internally in Gmod: https://wiki.facepunch.com/gmod/Global_Variables#misc

Reply:
Ah, alright 👍